src/General-Rules/RePointRule.class.st

```diff
@@ -0,0 +1,46 @@
+"
+This smell arise when a method contains a creation of an instance Point in this way : 
+
+Point x: y:
+
+prefer use x@y to create an instance of Point.
+"
+Class {
+	#name : 'RePointRule',
+	#superclass : 'ReNodeBasedRule',
+	#category : 'General-Rules-Migrated',
+	#package : 'General-Rules',
+	#tag : 'Migrated'
+}
+
+{ #category : 'manifest' }
+RePointRule class >> uniqueIdentifierName [
+	"This number should be unique and should change only when the rule completely change semantics"
+
+	^ 'PointRule'
+]
+
+{ #category : 'running' }
+RePointRule >> basicCheck: aNode [
+
+	aNode isMessage ifFalse: [ ^ false ].
+	^ #( #x:y: ) includes: aNode selector
```

Comment:
What if I have my own custom class and send an x:y: message to it - then it would lament as well (although it is not Point where the message is sent to) 

So I think the rule should check additionally for Point being the receiver of the message, no?

Reply:
I agree. And it would be nice to add a test on this exception